Apps/Common/src/AccessManagement/Administration/KeycloakUserAdmin.cs

```diff
@@ -13,31 +13,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //-------------------------------------------------------------------------
-namespace HealthGateway.Common.Delegates.AuthServer.Keycloak
+namespace HealthGateway.Common.AccessManagment.Administration
 {
     using System;
     using System.Collections.Generic;
-    using System.Diagnostics;
-    using System.Diagnostics.Contracts;
     using System.Net.Http;
     using System.Net.Http.Headers;
     using System.Net.Mime;
     using System.Text.Json;
     using System.Threading.Tasks;
 
-    using HealthGateway.Common.Delegates.AuthServer;
-    using HealthGateway.Common.Models.AuthServer;
+    using HealthGateway.Common.AccessManagment.Administration.Models;
     using HealthGateway.Common.Services;
     using Microsoft.Extensions.Configuration;
     using Microsoft.Extensions.Logging;
 
-
-
-
     /// <summary>
-    /// Class implementation of <see cref="IUserDelegate"/> that uses http REST to connect to Keycloak.
+    /// Class implementation of <see cref="IUserAdmin"/> that uses http REST to connect to Keycloak.
     /// </summary>
-    public class UserDelegate : IUserDelegate
+    public class KeycloakUserAdmin : IUserAdmin
```

Comment:
Same as above, class name and namespace does not indicate what kind of object this is.